Dockerfile

```diff
@@ -25,23 +24,51 @@ RUN yum --assumeyes install \
     npm \
     openssl \
     procps-ng \
-    python-pip \
-    python3.7 \ 
+    python36 \ 
+    python39 \ 
+    python39-pip \
     rsync \
-    sshuttle \
-    the_silver_searcher \
-    tmux \
     vim-enhanced \
     wget \
-    && yum clean all;
+    && microdnf clean all;
+
+RUN git clone --depth 1 https://github.com/junegunn/fzf.git ~/.fzf && \
+    ~/.fzf/install --all && \
+    rm -rf ~/.fzf
 
 
 ### Download the binaries
 # Anything in this image must be COPY'd into the final image, below
-FROM fedora:latest as builder
+FROM quay.io/app-sre/ubi8-ubi-minimal:8.4-200 as builder
 
 # jq is a pre-req for making parsing of download urls easier
-RUN dnf install -y jq rhash unzip
+RUN microdnf --assumeyes install \
+    gcc \
+    git \
+    jq \
+    make \
+    python39 \ 
+    python39-pip \
+    tar \
+    unzip \
+    virtualenv 
+
+RUN  virtualenv -p python3 /tmp/sshuttle && \
+     source /tmp/sshuttle/bin/activate && \
+     pip install sshuttle && \
+     cp /tmp/sshuttle/bin/sshuttle /usr/bin && \
+     deactivate && \
+     rm -rf /out/sshuttle
+
+RUN TMPPWD=${PWD} && \
+    mkdir -p /tmp/rhash && \
+    cd /tmp/rhash && \
+    git clone https://github.com/rhash/RHash.git && \
+    cd RHash  && \
+    ./configure && make install && \
+    cp rhash /usr/bin && \
+    cd ${TMPPWD} && \
+    rm -rf /tmp/rhash 
```

Comment:
currently this compiles but has an issue with the binary :( not sure what is the problem as of now.

failure is in https://github.com/openshift/ocm-container/pull/90/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L147

and the error is 
```
rhash: error while loading shared libraries: librhash.so.0: cannot open shared object file: No such file or directory
```